kong/plugins/http-log/schema.lua

```diff
@@ -2,6 +2,7 @@ return {
   fields = {
     http_endpoint = { required = true, type = "url" },
     method = { default = "POST", enum = { "POST", "PUT", "PATCH" } },
+    content_type = { default = "application/json", enum = { "application/json", "application/x-www-form-urlencoded" } },
```

Comment:
this should not be part of the http-logger, as it only does json. So here it should stay, but as an enum with only one possible value; `application/json`.

Reply:
Is it possible to override the 'application/json' default without forking and creating a custom HTTP-LOG plugin?